angular-ngc/tools/BUILD.bazel

```diff
@@ -34,6 +35,19 @@ ts_project(
 exports_files(
     [
         "karma.conf.js",
+        "test-setup.ts",
     ],
     visibility = ["//visibility:public"],
 )
+
+esbuild(
+    name = "prebuild_test_bootstrap",
```

Comment:
Lets not put "prebuild" into the name. Probably just "test-bootstrap" like the file name?

Reply:
Sure, done